src/components/forms/controls/DateTimePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
+
+import { classNames as cx, type ClassNameArgument } from '../../../../util/componentUtil.ts';
+
+import { DatePicker } from '../DatePicker/DatePicker.tsx';
+import { TimePicker, type Time } from '../TimePicker/TimePicker.tsx';
+
+import cl from './DateTimePicker.module.scss';
+
+
+type GenericProps = {
+  /** Whether this component should be unstyled. */
+  unstyled?: undefined | boolean,
+
+  /** An optional class name to be appended to the class list. */
+  className?: undefined | ClassNameArgument,
+  
+  /** A Date object to hold the date and time. */
+  date: Date | null,
```

Comment:
I think we can make this a required prop. The consumer can always initialize it to a default value like the current date/time.

Also for the other components.

Reply:
The problem is that the library expects it to be `Date | null`, I tried setting it as a plain `Date` but then we start to get type mismatches. Maybe setting it as `Date` on our code and then passing it down to the library with `as Date | null`?

Reply:
Documenting here our conversation: `Date | null` is a valid value for uninitialized calendar.